EIPS/eip-7923.md

```diff
@@ -145,21 +147,19 @@ index bf34fbee..477f969e 100644
 +        start_page = start_position >> 12
 +        end_page = end >> 12
 +
-+        gas = 0
-+
 +        for page in range(start_page, end_page + 1):
 +            if page not in self._memory.pages:
-+                gas += ALLOCATE_PAGE_COST
++                reason = f"Allocating page {page << 12}"
```

Comment:
Not sure if python does this automatically but I think this should be hex encoded?

Reply:
07bee6723f615078688c43e9f3f794bf8b35c68a